connect/templates/generator-maven-plugin/README.md

```diff
@@ -22,20 +20,17 @@
 
 | Name                     |                | Description                                                      | Possible Values           | Default Value                                                    |
 |--------------------------|----------------|------------------------------------------------------------------|---------------------------|------------------------------------------------------------------|
+| `project`                | **(required)** | The Maven project.                                               |                           | ${project}                                                       |
 | `targetPlatform`         | **(required)** | The target platform for which the templates should be generated. | `C7`, `C8`                | None                                                             |
 | `outputDirectory`        | **(optional)** | The directory where the generated templates should be saved.     | Any valid path            | `${project.build.directory/generated-sources/element-templates}` |
 | `inputValueNamingPolicy` | **(optional)** | The naming policy for the input value.                           | `EMPTY`, `ATTRIBUTE_NAME` | `EMPTY`                                                          |
+| `skip`                   | **(optional)** | Skip the execution of the plugin.                                | `true`, `false`           | `false`                                                          |
```

Comment:
Whats that for?
The plugin wont be executed automatically unless you specify the build phase, like in this example:

```
<plugin>
    <groupId>io.miragon.miranum.connect</groupId>
    <artifactId>element-templates-generator-maven-plugin</artifactId>
    <version>${miranum-connect.version}</version>
    <executions>
        <execution>
            <goals>
                <goal>generate</goal>
            </goals>
            <phase>generate-resources</phase>
        </execution>
    </executions>
    <configuration>
        <targetPlatforms>camunda7</targetPlatforms>
        <outputDirectory>${project.basedir}/src/main/resources/prozesse/element-templates</outputDirectory>
    </configuration>
</plugin>
```

Reply:
Good question. I am not sure it was already there.

Reply:
Shall we remove it? Wdyt?

Reply:
Mmh I would actually leave it because I think it is useful in some cases. But if you think otherwise, I can remove it. After all, you have more experience than I do 😃

Reply:
In my point of view it is best practices to disable it via maven standards. I agree with Andi, we should remove it.